### DISCUSSION TITLE
container windows hiden

### DISCUSSION THREAD

**Discussion by joCoutu** [microsoft/vscode-docker#3166] [Q&A] [answered]
FROM debian:stable

I duplicate the container windows. 
I click on hide container windows.

How can I reset the view like the default behaviour?

![Screenshot from 2021-08-22 05-35-28](https://user-images.githubusercontent.com/45253518/130350403-bda634d3-e925-4151-8475-f5ca1e3f046b.png)

**Comment by bwateratmsft** [comment, marked as answer]
You can run the command "View: Reset View Locations" and it will move everything back to their default spots. Otherwise, you can check "Containers" again and drag it back if you want to move just that view.

**Comment by joCoutu** [comment]
Tanks that fix my problem.
